diagrams/lib/main.dart

```diff
@@ -35,7 +37,20 @@ Future<Directory> prepareOutputDirectory() async {
   return directory;
 }
 
-Future<Null> main(List<String> arguments) async {
+Future<Null> main() async {
+  final List<String> arguments = window.defaultRouteName.length > 5
+      ? Uri.decodeComponent(window.defaultRouteName.substring(5)).split(' ')
+      : <String>[];
+  final ArgParser parser = new ArgParser();
+  parser.addMultiOption('category',
+      help: 'Only create the diagrams for the given category.');
```

Comment:
not sure we really need the help lines here, they're redundant with the "real" ones in the main file

Reply:
Yeah, I guess so: you can't really see them anyhow.